wwwroot/test/WFS/getCapabilities.xml

```diff
@@ -1,54 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<wfs:WFS_Capabilities version="2.0.0"
+<wfs:WFS_Capabilities version="1.1.0"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I replaced this test file because we don't actually support WFS version `2.0.0`

All WFS requests are sent with version `1.1.0`

### GetCapabilities

https://github.com/TerriaJS/terriajs/blob/e0f47a22e7725b9bb9d79eebeed8fab4f2a68778/lib/Models/Catalog/Ows/WebFeatureServiceCatalogItem.ts#L333-L337

### GetFeature

https://github.com/TerriaJS/terriajs/blob/e0f47a22e7725b9bb9d79eebeed8fab4f2a68778/lib/Models/Catalog/Ows/WebFeatureServiceCatalogItem.ts#L403-L412